src/commands/showprofile.ts

```diff
@@ -0,0 +1,42 @@
+import { fetchUser } from "../util/UserUtil/index.js";
+import { EmbedBuilder } from "discord.js";
+
+const __showProfile: InteractionHandlerPayloads.GuildChatInputCommand = {
+    name: "showprofile",
+    description: "Display your profile in a funni embed",
+    async execute(interaction) {
+        // fetch user profile
+        const user = await fetchUser(interaction.user.id);
+
+        // grab all the funni info and throw into a funni embed
+        const profileEmbed = new EmbedBuilder({
+            color: 0x900c3f,
+            title: user.displayName + "'s Profile",
+            thumbnail: {
+                url: interaction.member.displayAvatarURL(),
+            },
+            fields: [
+                {
+                    name: "Bio:",
+                    value: user.bio,
+                },
+                {
+                    name: "Wins:",
+                    value: user.wins.toString(),
+                },
+                {
+                    name: "Losses:",
+                    value: user.losses.toString(),
+                },
+                {
+                    name: "Points:",
+                    value: user.points.toString(),
+                },
+            ],
+        });
+
+        interaction.reply({ embeds: [profileEmbed], ephemeral: false }); // send the funni embed
```

Comment:
For consistency, please move the side-comment to be on top of the `interaction.reply(...;` line.